tests/unit/test_loader_collection.py

```diff
@@ -432,13 +432,14 @@ def test_missing_readme(populated_collection_root):
 @mock.patch("galaxy_importer.collection.CollectionLoader._build_docs_blob")
 def test_changelog_fail(_build_docs_blob, populated_collection_root, caplog):
     _build_docs_blob.return_value = {}
-
+    
```

Comment:
looks like some whitespace was added here

Reply:
Fixed this and the other lint errors.